hw_config.h

```diff
@@ -342,7 +342,7 @@
 # define INTERFACE_USART                1
 # define USBDEVICESTRING                "PX4 BL FMU v5.x"
 # define USBPRODUCTID                   0x0032
-# define BOOT_DELAY_ADDRESS             0x000001a0
+# define BOOT_DELAY_ADDRESS             0x00000200
```

Comment:
Not sure we can move this not still need it.

Reply:
I think this is actually a bugfix, I was not moving it, but just correcting the address where the boot delay signature is for real. I think that currently the bootloader is not finding the boot delay signature at all.

Reply:
@jlaitine is right. F7 has a slightly bigger vector table than F4
![image](https://user-images.githubusercontent.com/10188706/104205444-ae093580-542e-11eb-8366-6932da24c8bc.png)

Reply:
This same bug actually exists for some other targets as well, like stm32h7, the boot delay signature location has just been copy pasted from F4, even though the vectors size is different.

Reply:
Other then on the Solo, It has never been used. We should see if we can derive it from the HW lib's vector definitions and avoid this class of error.

Reply:
I've opened another issue for this topic https://github.com/PX4/PX4-Bootloader/issues/184 so we can focus on secure boot here. Let's keep this bug fix here as it is.